matchzoo/engine/base_model.py

```diff
@@ -122,10 +122,10 @@ def fit(
 
         See :meth:`keras.models.Model.fit` for more details.
 
-        :param x: input data
-        :param y: labels
-        :param batch_size: number of samples per gradient update
-        :param epochs: number of epochs to train the model
+        :param x: input data.
```

Comment:
I usually capitalize the first charactor when there's a period at the end of the line, unless the entire doc string fits in one line.